views/index.erb

```diff
@@ -47,13 +105,24 @@
 
 <!-- Add your Javascript to this script tag -->
 <script language="javascript">
-  console.log("I am Javascript!");
 
   var profile_values = document.getElementsByClassName('profile-value');
   var total_words = 0;
+  // Double check I can get length in the manner I want to.
+    // console.log(profile_values[1].innerHTML.split(" ").length)
+  // Get the number of profile values elements for my for loop.
+    // console.log(profile_values.length);
+  for (var i = 0; i < profile_values.length; i++) {
+    // Double check my for loop works as I want it to.
+      // console.log(profile_values[i].innerHTML.split(" ").length);
 
-  // TODO - Count the number of words in the profile.
-  // hint - look at what the innerHTML property provides.
+    // Add each loop to the total words variable.
+      total_words += profile_values[i].innerHTML.split(" ").length;
+    // Log to console.
+      // console.log(total_words)
+  };
 
   document.getElementById('word-count').innerHTML = total_words;
-</script>
\ No newline at end of file
+</script>
+<script src="/js/jquery-2.1.4.min.js"></script>
+<script src="/js/bootstrap.min.js"></script>
```

Comment:
You're missing a terminating newline character at the end of this file. Depending on your editor of choice, you could probably alter your configuration to ensure these are added for you automatically.